server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/clone/CloneSnapshotRequest.java

```diff
@@ -139,4 +142,27 @@ public String target() {
     public String source() {
         return this.source;
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
```

Comment:
This is kind of meh, we technically only need the `indices` and indices options fields here for the HLRC client. But then I dug into it and found that we add all these redundant fields for other requests as well so I kept it consistent here for now. I have an idea on how to make this nicer that I'd put in a follow up (could just use a `params` entry here).